compatibility/README.md

```diff
@@ -31,11 +32,31 @@ Version: [![Maven Central](https://maven-badges.herokuapp.com/maven-central/com.
 implementation group: 'com.foursoft.harness.compatibility', name: 'compatibility-core', version: 'VERSION'
 ```
 
-### Compatibility VEC 1.1.X to VEC 1.2.X
+### Compatibility VEC 1.2.X to VEC 2.0.X
+
+#### Maven
+
+```xml
+
+<dependency>
+    <groupId>com.foursoft.harness.compatibility</groupId>
+    <artifactId>compatibility-vec12to20</artifactId>
+    <version>VERSION</version>
+</dependency>
+```
+
+#### Gradle
+
+```groovy
+implementation group: 'com.foursoft.harness.compatibility', name: 'compatibility-vec12to20', version: 'VERSION'
+```
+
+### Compatibility VEC 1.1.3 to VEC 1.2.X
```

Comment:
In my opinion, the VEC 1.1 <-> 1.2 should come first. But since this MR doesn't contain that module, I would remove it again until it exists.